feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.html

```diff
@@ -2,6 +2,7 @@
   [cxFocus]="focusConfig"
   (esc)="close('Escape clicked')"
   class="cx-import-entries-dialog"
+  
```

Comment:
I think this line insertion can be deleted right?

Reply:
Yes